gulpfile.js

```diff
@@ -254,23 +256,48 @@ gulp.task('generate-plunks', function() {
   return gulpFile(plunks, {src: true}).pipe(gulp.dest('demo/src/public/app/components'));
 });
 
-gulp.task('clean:demo', function() { return del('demo/dist'); });
+gulp.task('clean:demo', function() {
+
+  spawn('npm', ['uninstall', '@ng-bootstrap/ng-bootstrap'], {stdio: 'inherit'}).on('error', function(err) {
+    console.log('Failed to run npm uninstall @ng-bootstrap/ng-bootstrap : ' + err);
+  });
+  return del('demo/dist');
+
+});
 
 gulp.task('clean:demo-cache', function() { return del('.publish/'); });
 
-gulp.task(
-    'demo-server', ['generate-docs', 'generate-plunks'],
-    shell.task([`webpack-dev-server --port ${docsConfig.port} --config webpack.demo.js --inline --progress`]));
 
-gulp.task(
-    'build:demo', ['clean:demo', 'generate-docs', 'generate-plunks'],
-    shell.task(['webpack --config webpack.demo.js --progress --profile --bail'], {env: {MODE: 'build'}}));
+gulp.task('demo-server', ['clean:demo', 'generate-docs', 'generate-plunks', 'angular-cli-jit'], function() {
+  spawn('ng', ['serve', '--port', docsConfig.port], {stdio: 'inherit'}).on('error', function(err) {
+    console.log('Failed to run child process : ' + err);
```

Comment:
It might be good to do `console.error` here.
Remove the extra space before `:`

One can also use a template literal here, i.e.
```js
`Failed to run child process: ${err}`
```